samples/snippets/fsharp/lang-ref-1/snippet1901.fs

```diff
@@ -1,2 +1,18 @@
-type Point = { x : float; y: float; z: float; }
-type Customer = { First : string; Last: string; SSN: uint32; AccountNumber : uint32; }
\ No newline at end of file
+// Labels are separated by semicolons when defined on the same line.
+type Point = { x: float; y: float; z: float; }
+
+// You can define labels on their own line with a semicolon.
+type Customer = 
+    { First: string
+      Last: string
+      SSN: uint32
+      AccountNumber: uint32; }
+
+// A struct record.
+// The way you format the braces doesn't matter.
+[<Struct>]
+type StructPoint = { 
```

Comment:
In the code samples, record declarations should use format

```fsharp
type X = 
    { A : int
      B : string }
```
rather than 
```
type X = {
    A : int
    B : string
}
```
Note the two inconsistent styles here for ``Customer`` and ``StructPoint`` particularly